vexim/adminuserchangesubmit.php

```diff
@@ -30,10 +30,15 @@
   }
   if (isset($_POST['on_forward'])) {
     $_POST['on_forward'] = 1;
-    if(!filter_var($_POST['forward'], FILTER_VALIDATE_EMAIL)) {
-      header ("Location: adminuser.php?invalidforward=".htmlentities($_POST['forward']));
-      die;
+    $forwardto=explode(",",$_POST['forward']);
```

Comment:
That's what it did until now: overwrote whatever was in the text field with a single chosen email address. Implementing that in the front-end the same fashion won't even change the funtionality, just make it more obviuos.
One thing I'm a bit worried about is that if a user chooses an address accidentally, they can select an empty value again before submitting, thus leaving the contents of the forward field intact. If we make the selection propagate into the text field immediately, it will be possible to mess up the contents of one's `forward` field by accident.
Perhaps adding the address after a comma is a better approach. Alternatively, perhaps if the initial contents of the field aren't empty, there should be a Yes/No dialog?